src/App/Resources/AppResources.resx

```diff
@@ -954,6 +954,9 @@ Scanning will happen automatically.</value>
   <data name="UpdateKey" xml:space="preserve">
     <value>You cannot use this feature until you update your encryption key.</value>
   </data>
+  <data name="EncryptionKeyMigrationRequired" xml:space="preserve">
+    <value>Encryption key migration required. Please login through the web vault to update your encryption key.</value>
+  </data>
```

Comment:
⛏️ I think the string `Encryption key migration required` could be something we could use and given that this key is here we couldn't use the same key again. I'd use another key that indicates that it has a lengthier description. I tend to use `DescriptionLong` as a suffix given that is highly unlikely something like that could be used actually as a resource in the app.
```suggestion
  <data name="EncryptionKeyMigrationRequiredDescriptionLong" xml:space="preserve">
    <value>Encryption key migration required. Please login through the web vault to update your encryption key.</value>
  </data>
```